bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -241,7 +253,6 @@ public BigtableAsyncAdmin(BigtableAsyncConnection asyncConnection) throws IOExce
     return CompletableFuture.completedFuture(!disabledTables.contains(tableName));
   }
 
-  @Override
   public CompletableFuture<TableDescriptor> getTableDescriptor(TableName tableName) {
```

Comment:
If this method is removed from the HBase interface, please remove this method, and have all uses of this method use `getDescriptor`